src/utils.js

```diff
@@ -1,22 +1,17 @@
+import { join } from 'path';
 import { statSync } from 'fs';
 
 // @ts-ignore
 import arrify from 'arrify';
 
-const UNESCAPED_GLOB_SYMBOLS_RE = /(\\?)([()*?[\]{|}]|^!|[!+@](?=\())/g;
-
 /**
  * @param {string|string[]} files
  * @param {string} context
  * @returns {string[]}
  */
 export function parseFiles(files, context) {
-  return arrify(files).map(
-    (/** @type {string} */ file) =>
-      `${replaceBackslashes(context).replace(
-        UNESCAPED_GLOB_SYMBOLS_RE,
-        '\\$2'
-      )}/${replaceBackslashes(file)}`
+  return arrify(files).map((/** @type {string} */ file) =>
+    replaceBackslashes(join(context, file))
```

Comment:
Glob should always have `/`

Reply:
In fact it was like this, they may not work when we have a folder like the test `[symbols]`.
I'm taking the regex out of context, because it doesn't really make sense

https://github.com/webpack-contrib/eslint-webpack-plugin/blob/master/src/utils.js#L6
https://github.com/mrmlnc/fast-glob/blob/master/src/utils/path.ts#L6

Reply:
:+1:

Reply:
This does not apply here